tests/lib/MVC/Symfony/EventListener/SiteAccessMatchListenerTest.php

```diff
@@ -155,6 +182,38 @@ public function testOnKernelRequestSerializedSAWithCompoundMatcher()
         $this->assertFalse($request->attributes->has('serialized_siteaccess'));
     }
 
+    public function testOnKernelRequestSerializedSAWithMatcherInMatcherRegistry(): void
```

Comment:
CS: unhandled exceptions
```suggestion
    /**
     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\NotFoundException
     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\InvalidArgumentException
     */
    public function testOnKernelRequestSerializedSAWithMatcherInMatcherRegistry(): void
```